src/App.jsx

```diff
@@ -14,45 +16,124 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_BY_ALPHABET = 'alphabet';
+const SORT_BY_LENGTH = 'length';
+
+function getPrepearedGoods(goods, sortField, sortRevers) {
+  const prepearedGoods = [...goods];
+
+  if (sortField === SORT_BY_ALPHABET) {
+    prepearedGoods.sort((a, b) => {
+      switch (sortRevers) {
+        case true:
+          return b.localeCompare(a);
+        case false:
+          return a.localeCompare(b);
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (sortField === SORT_BY_LENGTH) {
+    prepearedGoods.sort((a, b) => {
+      switch (sortRevers) {
+        case true:
+          return b.length - a.length;
+        case false:
+          return a.length - b.length;
+        default:
+          return 0;
+      }
+    });
+  }
```

Comment:
it is can be one sort with switch case  by sortField
no need to check isReverse here.
```suggestion

    prepearedGoods.sort((a, b) => {
      switch (sortField) {
        case SORT_BY_LENGTH:
          return a.length - b.length;
        case SORT_BY_ALPHABET:
          return a.localeCompare(b);
        default:
          return 0;
      }
    });
```